ansible/roles/nginx/templates/nginx.conf.j2

```diff
@@ -90,9 +90,25 @@ http {
             return 301 https://github.com/apache/incubator-openwhisk-client-swift/releases/download/0.2.3/starterapp-0.2.3.zip;
         }
 
+{% if openwhisk_cli.installation_mode == "remote_redirect" %}
+        # redirect requests for specific binaries to the matching one from the {{ openwhisk_cli_tag }} openwhisk-cli release.
+  {% for os in cli_os %}
+     {% for arch in cli_arch %}
```

Comment:
This concerns me given changes I made in #3133 to introduce cli_os_arch.  It will be a simple update, depending on which goes first.